components/callbacks/executors.go

```diff
@@ -143,7 +143,7 @@ func (e taskExecutor) executeInvocationTask(
 	defer cancel()
 
 	result := invokable.Invoke(callCtx, ns, e, task)
-	saveErr := e.saveResult(callCtx, env, ref, result)
+	saveErr := e.saveResult(ctx, env, ref, result)
```

Comment:
does this ctx have any timeout/deadline?

Reply:
That's the task context, it shouldn't have a deadline AFAIK.